src/sections/Meshery/Meshery-integrations/Individual-Integrations/relatedIntegration.js

```diff
@@ -89,10 +91,10 @@ const RelatedIntegration = ({ category }) => {
       <Slider {...settings}>
         {
           IntegrationList.map((item, index) => {
-            if (item.category === category){
+            if (item.category === category && item.status != "InProgress"){
               return (
                 <div key={index}>
-                  <Link to={`/service-mesh-management/meshery/integrations/${item.name.toLowerCase()}`}>
+                  <Link to={`/service-mesh-management/meshery/integrations/${item.name.replaceAll(" ", "-").toLowerCase()}`}>
```

Comment:
These are not needed after the name update.
Update at all rhe places.

Reply:
Ok, so you are using the integrationList here and not the data from graphql query, huh?
This has to change as well, maybe in the next PR